docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -0,0 +1,170 @@
+
+Pre-existing raw tables, SCD tables and "unnested" tables will always be left untouched. You can delete these at your convenience.
+Each destination version is managed separately, so if you have multiple destinations, they all need to be upgraded one by one.
+
+Versions are tied to the destination. When you update the destination, **all connections tied to that destination will be sending data in the Destinations V2 format**. For upgrade paths that will minimize disruption to existing dashboards, see:
+* [Upgrading Connections One by One with Dual-Writing](#upgrading-connections-one-by-one-with-dual-writing)
+* [Testing Destinations V2 on a Single Connection](#testing-destinations-v2-for-a-single-connection)
+* [Upgrading as a User of Raw Tables](#upgrading-as-a-user-of-raw-tables)
+* [(OSS Only) Rolling back to Legacy Normalization](#oss-only-rolling-back-to-legacy-normalization)
+ 
+## Advanced Upgrade Paths
+
+### Upgrading Connections One by One with Dual-Writing
+
+Dual writing is a method employed during upgrades where new incoming data is written simultaneously to both the old and new systems, facilitating a smooth transition between versions. We recommend this approach for connections where you are especially worried about breaking changes or downtime in downstream systems.
+
+#### Steps to Follow for All Sync Modes
+
+1. **[Open Source]** Update the default destination version for your workspace to a [Destinations V2 compatible version](#destinations-v2-effective-versions). This sets the default version for any newly created destination.
+
+![Upgrade your default destination version](assets/airbyte_version_upgrade.png)
+
+2. Create and configure a new destination connecting to the same database as your existing destination except for `Default Schema`, which you should update to a new value to avoid collisions.
+
+![Create a new destination](assets/airbyte_dual_destinations.png)
+
+3. Create a new connection leveraging your existing source and the newly created destination. Match the settings of your pre-existing connection.
+4. If the streams you are looking to replicate are in **full refresh** mode, enabling the connection will now provide a parallel copy of the data in the updated format for testing. If any of the streams in the connection are in an **incremental** sync mode, follow the steps below before enabling the connection.
+
+#### Additional Steps for Incremental Sync Modes
+
+These steps allow you to dual-write for connections incrementally syncing data without re-syncing historical data you've already replicated:
+
+1. Copy the raw data you've already replicated to the new schema being used by your newly created connection. You need to do this for every stream in the connection with an incremental sync mode. Sample SQL you can run in your data warehouse:
+
+```mysql
+CREATE TABLE {new_schema}.raw_{stream_name} AS 
+SELECT * 
+FROM {old_schema}.raw_{stream_name};
+```
```

Comment:
Evan TODO:
* We'll need to make the new table first with the right columns and types.  This query will differ per destination
* We will load each column by name to handle the rename (_emitted_at -> _extracted_at)